imports/state/epics/resend-verification.js

```diff
@@ -0,0 +1,28 @@
+import { Accounts } from 'meteor/accounts-base'
+import { Meteor } from 'meteor/meteor'
+import { Subject } from 'rxjs/Subject'
+import { RESEND_VERIFICATION, ERROR, SUCCESS } from '../../ui/unverified-warning/unverified-warning.actions'
+
+import 'rxjs/add/operator/switchMap'
+import 'rxjs/add/operator/filter'
+
+export const resendVerification = action$ =>
+  action$.ofType(RESEND_VERIFICATION)
+    .filter(() => !Meteor.userId())
+    .switchMap(({userId}) => {
```

Comment:
Instead of receiving `userId`, call `Meteor.userId()` just below to get it without passing it all the way here